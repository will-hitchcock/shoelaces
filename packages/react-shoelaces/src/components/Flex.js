import styled from 'styled-components'
import { flex } from 'shoelaces'

export const Flex = styled.div`${flex}`

export default Flex
