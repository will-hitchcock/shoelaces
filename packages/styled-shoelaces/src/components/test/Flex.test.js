import React from 'react';
import 'jest-styled-components';
import { shallow } from 'enzyme';
import Flex from '../Flex';

describe('<Flex />', () => {
  it('should render a <Flex />', () => {
    expect(shallow(<Flex />)).toMatchSnapshot();
  });
});
