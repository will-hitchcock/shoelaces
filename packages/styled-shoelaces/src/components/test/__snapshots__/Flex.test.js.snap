// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Flex /> should render a <Flex /> 1`] = `
.c0 {
  -webkit-align-content: flex-start;
  -ms-flex-line-pack: start;
  align-content: flex-start;
  -webkit-align-items: stretch;
  -webkit-box-align: stretch;
  -ms-flex-align: stretch;
  align-items: stretch;
  -webkit-align-self: auto;
  -ms-flex-item-align: auto;
  align-self: auto;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-flex-wrap: nowrap;
  -ms-flex-wrap: nowrap;
  flex-wrap: nowrap;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  -webkit-order: 0;
  -ms-flex-order: 0;
  order: 0;
}

<div
  className="c0"
/>
`;
